src/NUnitFramework/TestBuilder.cs

```diff
@@ -60,51 +64,19 @@ public static TestSuite MakeFixture(object fixture)
             return suite;
         }
 
-        public static TestSuite MakeFixture(List<Type> types)
-        {
-            // following the pattern found in DefaultTestAssemblyBuilder
-            var fixtures = new List<Test>();
-            var defaultSuiteBuilder = new DefaultSuiteBuilder();
-            foreach (var type in types)
-            {
-                var typeInfo = new TypeWrapper(type);
-                var test = defaultSuiteBuilder.BuildFrom(typeInfo);
-                fixtures.Add(test);
-            }
-
-            var assembly = AssemblyHelper.Load("nunit.testdata");
-            var assemblyPath = AssemblyHelper.GetAssemblyPath(assembly);
-            TestSuite testSuite = new TestAssembly(assembly, assemblyPath);
-
-            var treeBuilder = new NamespaceTreeBuilder(testSuite);
-            treeBuilder.Add(fixtures);
-
-            return treeBuilder.RootSuite;
-        }
-
         public static TestSuite MakeParameterizedMethodSuite(Type type, string methodName)
         {
             var suite = MakeTestFromMethod(type, methodName) as TestSuite;
             Assert.NotNull(suite, "Unable to create parameterized suite - most likely there is no data provided");
             return suite;
         }
 
-        public static TestSuite MakeParameterizedMethodSuite(object fixture, string methodName)
```

Comment:
The hard part with reviewing commits this big is that it seems like this refactor will be a very cross-cutting change, but I'm all for streamlining this API. 😃